04_testing/max_pin_adjustment.R

```diff
@@ -44,7 +44,7 @@ df_sectors <- read_csv(
 
 df_msna <- read_csv(
   file.path(
-    dirname(getwd()),
+    file_paths$agg_dir,
```

Comment:
Ah you've already added the MSNA data to the Sharepoint, great! We just need to change the rest of the code to read the MSNA data from there.